lib/turnout/maintenance_page.rb

```diff
@@ -11,14 +12,14 @@ def self.best_for(env)
 
       all_types = all.map(&:media_types).flatten
       best_type = request.best_media_type(all_types)
-
-      best = all.find { |page| page.media_types.include? best_type }
+      best = all.find { |page| page.media_types.include?(best_type) && File.exist?(page.new.path) }
```

Comment:
The `File.exist?` call is always returning `true` because the paths always exist in the gem. I think we'll have to first check if the custom path exists and if none of those are found then go with the default page in the gem. Does that make sense? If you have time to make this change, that would be great. Otherwise I'll see if I can get to it.